test/case/infix_containers/container_bridge/test.py

```diff
@@ -84,18 +84,18 @@
             }
         })
 
-    with test.step(f"Verify {NAME} container has started"):
+    with test.step("Verify container 'web-docker0' has started"):
         c = infamy.Container(target)
         until(lambda: c.running(NAME), attempts=10)
 
-    with test.step(f"Verify {NAME} container responds"):
-        _, hport = env.ltop.xlate("host", "data")
-        url = infamy.Furl(URL)
+    _, hport = env.ltop.xlate("host", "data")
+    url = infamy.Furl(URL)
 
-        with infamy.IsolatedMacVlan(hport) as ns:
-            ns.addip(OURIP)
+    with infamy.IsolatedMacVlan(hport) as ns:
+        ns.addip(OURIP)
+        with test.step("Verify host:data can ping 10.0.0.2"):
```

Comment:
Maybe "Verify basic DUT connectivity, host:data can ping DUT 10.0.02" or something?  It's not for the container, so not to confuse the reader what this step is for.